status.iss

```diff
@@ -0,0 +1,56 @@
+#define   Name       "Status"
+#define   Version    "0.0.1"
+#define   Publisher  "Status.im"
+#define   URL        "https://status.im"
+#define   ExeName    "Status.exe"
+
+[Setup]
+
+; Generated from Tools -> Generate GUID
+AppId={{F3E2EDB6-78E8-4539-9C8B-A78F059D8647}}
+
+AppName={#Name}
+AppVersion={#Version}
+AppPublisher={#Publisher}
+AppPublisherURL={#URL}
+AppSupportURL={#URL}
+AppUpdatesURL={#URL}
+
```

Comment:
I would suggest to add
`WizardStyle=modern`
to make it looking more or less like Windows 8/10 UI